MANIFEST.in

```diff
@@ -0,0 +1,7 @@
+recursive-include * *.md
+recursive-include * *.rst
+recursive-include * *.txt
+
+recursive-include tests *
+recursive-exclude * __pycache__
+recursive-exclude * *.py[co]
```

Comment:
This file is necessary to allow installation from source distributions (because setup.py depends upon README.md being present!)